build.sh

```diff
@@ -16,10 +16,10 @@ curl -L --create-dirs https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scr
 chmod +x $GET_DOTNET_SCRIPT
 
 # Execute install script
-$GET_DOTNET_SCRIPT
+$GET_DOTNET_SCRIPT --channel "preview" --version Latest --install-dir "$DOTNET_INSTALL_DIR" --no-path
```

Comment:
Should we freeze the version when RC2 is released? (which is scheduled for [this week or next one](https://blogs.msdn.microsoft.com/dotnet/2016/05/06/net-core-rc2-improvements-schedule-and-roadmap/))